cmds/tui.c

```diff
@@ -162,10 +162,11 @@ static const char *help[] = {
 	"q             Quit",
 };
 
-#define NUM_GRAPH_FIELD 6
+#define NUM_GRAPH_FIELD 9
 
 static const char *graph_field_names[NUM_GRAPH_FIELD] = { "TOTAL TIME", "TOTAL AVG", "TOTAL MIN",
-							  "TOTAL MAX",	"SELF TIME", "ADDRESS" };
+							  "TOTAL MAX",	"SELF TIME", "SELF AVG",
+							  "SELF MIN",	"SELF MAX",  "ADDRESS" };
```

Comment:
Please fix the indentation by adding a comma after "ADDRESS".  It's not your fault but I'd like to change it here as well.